manifests/keyring.pp

```diff
@@ -0,0 +1,68 @@
+# @summary Manage GPG keyrings for apt repositories
+#
+# @example Install the puppetlabs apt source with keyring.
```

Comment:
can you regenerate the REFERENCE.md in this PR so we can check if this renders correctly? Indent for code examples is sometimes... tricky :D

Reply:
As suggested, generated the new REFERENCE.md file.

Reply:
Hey!  It looks like the REFERENCE was build with an older version of puppet-strings.  Can you try to `bundle update` and re-run `bundle exec rake strings:generate:reference`.  This should remove a lot of changes in REFERENCE.md introduced in the PR (mostly anchors in the table of contents).

If nothing changed, you might have hit a nasty bundler bug I have seen a few times…  Move your `Gemfile.lock` away, and run `bundle update` / `bundle exec rake strings:generate:reference` one more time. If it helps, I would be interested in your old `Gemfile.lock` to try to reproduce the issue and better understand what is wrong.